<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProfileDialog</class>
 <widget class="QDialog" name="ModifyProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify your profile</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password:</string>
   </property>
   <property name="buddy">
    <cstring>newPasswordLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Here you can modify your profile picture and/or your password.</string>
   </property>
  </widget>
  <widget class="QLabel" name="repeatPasswordLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat Password:</string>
   </property>
   <property name="buddy">
    <cstring>repeatPasswordLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeatPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageStatusLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Image profile:</string>
   </property>
   <property name="buddy">
    <cstring>imageLabel</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../notepad.qrc">:/images/default_profile_pic.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="newImageButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose new image...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../notepad.qrc"/>
 </resources>
 <connections/>
</ui>
